src/appleseed.shaders/src/appleseed/as_subsurface.osl

```diff
@@ -74,14 +76,16 @@ shader as_subsurface
         float max = 1.0,
         string label = "Subsurface Weight",
         string page = "Subsurface",
-        int divider = 1
+        int divider = 1,
+        int as_max_param_id = 3
```

Comment:
What's the rationale with the param ID numbering? Are you leaving space for new params in between existing ones?

In that case, I would suggest a step size of 10 instead of 2. I would also suggest not making an exception for the first two params (that currently have IDs 0 and 1, i.e. they don't follow the scheme used for the next params).

Reply:
IDs are the same that 3ds max plugin assigns to these parameters right now. So it should not be any difference between using provided IDs or generated by max.
All the new parameters should have ID greater than the greatest number in the shader.

Reply:
Also output parameters have the same ID in max since they are all under one UI parameter (combobox)

Reply:
Understood!